azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -16,6 +16,18 @@
 
 package azkaban.project;
 
+import azkaban.database.AbstractJdbcLoader;
```

Comment:
Can we have another module like tools or something where people can check in such tools? Not merging does not sound like the most optimal approach.
(Something to think about)